holidays/countries/british_virgin_islands.py

```diff
@@ -0,0 +1,205 @@
+        * [2019](https://web.archive.org/web/20250315162108/https://www.bvi.gov.vg/media-centre/2019-public-holidays)
+        * [2020](https://web.archive.org/web/20250111172241/https://www.bvi.gov.vg/media-centre/2020-public-holidays)
+        * [2021](https://web.archive.org/web/20250417222246/http://www.bvi.gov.vg/media-centre/2021-public-holidays-revised)
+        * [2022](https://web.archive.org/web/20250420005523/https://www.bvi.gov.vg/media-centre/revised-2022-public-holidays)
+        * [2023](https://web.archive.org/web/20250111170511/https://bvi.gov.vg/media-centre/updatedofficialholidays20231)
+        * [2024](https://web.archive.org/web/20250502194703/https://www.bvi.gov.vg/media-centre/cabinet-approves-2024-public-holidays)
+        * [2025](https://web.archive.org/web/20250124122752/https://bvi.gov.vg/media-centre/cabinet-approves-2025-holidays)
+    """
+
+    country = "VG"
+    default_language = "en_VG"
+    # %s (observed).
+    observed_label = tr("%s (observed)")
+    supported_languages = ("en_US", "en_VG")
+    start_year = 1967
+
+    def __init__(self, *args, **kwargs):
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        StaticHolidays.__init__(self, BritishVirginIslandsStaticHolidays)
+        kwargs.setdefault("observed_rule", SAT_SUN_TO_NEXT_MON)
+        kwargs.setdefault("observed_since", 2000)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_observed(self._add_new_years_day(tr("New Year's Day")))
+
+        self._add_holiday_1st_mon_before_mar_7(
+            # Lavity Stoutt's Birthday.
+            tr("The Anniversary of the Birth of Hamilton Lavity Stoutt")
+        )
+
+        if self._year <= 2020:
+            # Commonwealth Day.
+            self._add_holiday_2nd_mon_of_mar(tr("Commonwealth Day"))
+
+        # Good Friday.
+        self._add_good_friday(tr("Good Friday"))
+
+        # Easter Monday.
+        self._add_easter_monday(tr("Easter Monday"))
+
+        # Whit Monday.
+        self._add_whit_monday(tr("Whit Monday"))
+
+        # Sovereign's Birthday.
+        name = tr("Sovereign's Birthday")
+        if self._year >= 2019:
+            sovereign_birthday_dates = {
+                2019: (JUN, 7),
+                2023: (JUN, 16),
+            }
+            if self._year in sovereign_birthday_dates:
+                self._add_holiday(name, sovereign_birthday_dates[self._year])
+            else:
+                self._add_holiday_2nd_fri_of_jun(name)
+        else:
+            self._add_holiday_2nd_sat_of_jun(name)
+
+        name = (
+            # Virgin Islands Day.
+            tr("Virgin Islands Day")
+            if self._year >= 2021
+            # Territory Day.
+            else tr("Territory Day")
+            if self._year >= 1978
+            # Colony Day.
+            else tr("Colony Day")
+        )
```

Comment:
This style is something we tried to avoid for a while. I guess it's time to start accepting it.